packages/number-field/test/number-field.test.ts

```diff
@@ -73,10 +73,14 @@ describe('NumberField', () => {
     describe('receives input', () => {
         it('without language context', async () => {
             const el = await getElFrom(Default({ value: 1337 }));
+            const inputElement: HTMLInputElement =
```

Comment:
I _think_ this is the same as `el.focusElement`. If so, would you mind swapping that?